File: search and sort/sort/selection_sort.cpp
def selection_sort(a,n):
    for i in range(0,n-2):
        imin=i
        for j in range(i+1,n):
            if a[j]<a[imin]:
                imin=j
        temp=a[i]
        a[i]=a[imin]
        a[imin]=temp
    return a


a=list(map(int,input().split()))
n=len(a)
print(selection_sort(a,n))

File: basic/selection_sort.py
a=[64, 34, 25, 12, 22, 11, 90,1,2,100,19]

for i in range(len(a)):
    for j in range(i+1,len(a)):
        if a[j]<a[i]:
            a[j],a[i]=a[i],a[j]
print(a)
